domains/POAS/events/lpa-updated/schema.json

```diff
@@ -9,10 +9,69 @@
       "description": "The UID of the LPA",
       "pattern": "M(-[A-Z0-9]{4}){3}"
     },
+    "actorUID": {
+      "type": "string",
+      "description": "The UID of the actor making the change",
+      "pattern": "urn:opg:poas:makeregister:users:([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12})"
+    },
     "changeType": {
       "type": "string",
-      "description": "The type of change to the LPA"
+      "description": "The type of change to the LPA",
+      "enum": ["CREATED", "CERTIFICATE_PROVIDER_SIGN", "ATTORNEY_SIGN", "TRUST_CORPORATION_SIGN"]
     }
   },
-  "required": ["uid", "changeType"]
+  "allOf": [
```

Comment:
This is gross but doing an `if changeType == CREATED then required = ["uid", "changeType"] else required = ["uid", "actorUID", "changeType"]` was skipping validation on the `changeType` itself. This way catches all validations together.